app/controllers/contact/govuk/problem_reports_controller.rb

```diff
@@ -1,9 +1,9 @@
 class Contact::Govuk::ProblemReportsController < ContactController
-  DONE_OK_TEXT = "<h2>Thank you for your help.</h2> " \
-    "<p>If you have more extensive feedback, " \
-    "please visit the <a href='/contact'>contact page</a>.</p>".freeze
-  DONE_INVALID_TEXT = "<h2>Sorry, we're unable to send your message as you haven't given us any information.</h2> " \
-    "<p>Please tell us what you were doing or what went wrong.</p>".freeze
+  DONE_OK_TEXT = "<h1 class='govuk-heading-l'>Thank you for your help.</h1> " \
```

Comment:
Should this be a h1? I'm aware that multiple h1's on a page isn't necessarily a problem but I'm not clear on why this has been changed.

Reply:
In a no-js experience, the only thing the page contains is the markup listed here, which means a heading and two paragraphs (pictured above). In such case, I don't see why we would mark it as level 2 – that was my thinking.

Reply:
Probably the crop is too tight, but we only have the heading right before the content and the footer down below.

Reply:
Ahaaaaa. I had it in my head that this was part of the feedback gem component 🤦  Approving now.